### DISCUSSION TITLE
本地磁盘扫描不到路径

### DISCUSSION THREAD

**Discussion by i-ices** [AlistGo/alist#602] [Q&A] [answered]
我在openwrt上接了一个外置的硬盘，格式为ext4，并且已经挂载上去了，
![image](https://user-images.githubusercontent.com/69052395/155276637-65cb6352-28b7-4a2f-b9a2-305ccc589452.png)

在本地终端上可以访问到该文件夹

![image](https://user-images.githubusercontent.com/69052395/155276721-5ef7e104-7fb1-4ad4-9ef0-74de2d109eac.png)

在alist上扫描不到该文件夹

![image](https://user-images.githubusercontent.com/69052395/155276542-6f5ed1ac-1667-466a-b3f2-10dc4e32370e.png)
![image](https://user-images.githubusercontent.com/69052395/155276781-e83d126a-80f3-4d5a-88bc-b2a0a8b9998e.png)

我把路径切换成根路径后也没有看到该文件夹

![image](https://user-images.githubusercontent.com/69052395/155277126-ffd26fae-564e-4365-a7fe-d89da4b34fb4.png)
![image](https://user-images.githubusercontent.com/69052395/155277157-8773782d-2223-4069-994f-9f9135f6663b.png)

采用的是docker运行，运行语句参照文档

![image](https://user-images.githubusercontent.com/69052395/155277290-2d4863b2-6e12-43cf-89b9-1b7f23a48194.png)

我希望是能把存储地址改到我那个外接硬盘，尝试了很久还是不行

**Comment by xhofe** [comment, marked as answer]
docker 只能访问到容器内的文件。。按照你这样映射要挂载宿主机`/ices/alist/data`的话，跟路径应该填`/opt/alist/data/data`

**Reply by i-ices** [reply]
成功解决，谢谢

**Reply by Deusxxxx** [reply]
不是很理解，请问可以详细讲一下吗？我用 ubuntu 系统，docker 运行
 sudo docker run -d --restart=always -v /etc/alist:/opt/alist/data -p 5244:5244 -e PUID=0 -e PGID=0 -e UMASK=022 --name="alist"
宿主机有一块机械硬盘，想把这块机械硬盘挂到 alist 中去，但是实现不了

**Reply by wu-zeee** [reply]
我也不理解，能具体说说吗，我是在1palenl面板中安装的

**Comment by yuebinliu** [comment]
添加一个影射，把你要访问的目录映射进容器目录内

**Reply by yuebinliu** [reply]
docker run -d --restart=always -v /yourdata1:/opt/alist/data  -v /youdata2:/opt/alist/data/data2 -p 5244:5244 -e PUID=0 -e PGID=0 -e UMASK=022 --name="alist" 

这样运行下来，你在后台添加本地存储，譬如想加载youdata2，根文件夹路径就是/opt/alist/data/data2
即
yourdata1对应/opt/alist/data
/youdata2对应/opt/alist/data/data2

其实你也可以映射到根目录，譬如
/yourdata:/
这样，后台根文件夹路径直接填入/
